strum_macros/src/macros/enum_count.rs

```diff
@@ -2,13 +2,15 @@ use proc_macro2::TokenStream;
 use quote::quote;
 use syn::{Data, DeriveInput};
 
-use crate::helpers::non_enum_error;
+use crate::helpers::{non_enum_error, HasTypeProperties};
 
 pub(crate) fn enum_count_inner(ast: &DeriveInput) -> syn::Result<TokenStream> {
     let n = match &ast.data {
         Data::Enum(v) => v.variants.len(),
         _ => return Err(non_enum_error()),
     };
+    let type_properties = ast.get_type_properties()?;
+    let strum = type_properties.get_crate_path();
```

Comment:
Can you find a more descriptive name here than strum. Maybe `strum_module_path` or something

Reply:
Changed to
```rust
let strum_module_path = type_properties.crate_module_path();
```